.github/workflows/revert-pr.yaml

```
@@ -0,0 +1,91 @@
+      - name: Add Revert Label
+        id: add-revert-label
+        if: ${{ steps.fetch-pr.outputs.PR_NUMBER && github.event.workflow_run.conclusion == 'failure' }}
+        run: |
+          PR_NUMER=${{ steps.fetch-pr.outputs.PR_NUMBER }}
+          echo "Adding revert label to PR: $PR_NUMBER"
+          gh pr edit "$PR_NUMBER" --add-label "revert"
+
+  create-revert-pr:
+    name: Automated PR revert
+    needs: detect-failure
+    if: ${{ needs.fetch-pr.outputs.PR_NUMBER && github.event.workflow_run.conclusion == 'failure' }}
```

actionlint (1.7.4)

property "fetch-pr" is not defined in object type {detect-failure: {outputs: {pr_number: string}; result: string}}

(expression)